gcc/config/riscv/riscv.cc

```diff
@@ -4569,7 +4569,7 @@ riscv_legitimize_move (machine_mode mode, rtx dest, rtx src)
 
       temp_reg = gen_reg_rtx (word_mode);
       zero_extend_p = (LOAD_EXTEND_OP (mode) == ZERO_EXTEND);
-      emit_insn (gen_extend_insn (temp_reg, src, word_mode, mode, 
+      emit_insn (gen_extend_insn (temp_reg, src, word_mode, mode,
```

Comment:
It seems that the auto-trimming whitespace in my VScode create this issue. I fix it https://github.com/riscv-collab/riscv-gcc/pull/334/commits/15ba09286c0af8f6e23a2f9ece7a7a716002556b here.